dot_lpdev_cmds.sh

```diff
@@ -261,7 +261,7 @@ function __lpdev_geth_run {
       -rpccorsdomain "*" -wsorigins "*" \
       -rpcapi 'personal,account,eth,web3,net' \
       -wsapi 'personal,account,eth,web3,net' \
-      -targetgaslimit 6700000 -unlock $gethMiningAccount --password <(echo "") -mine &>>$nodeBaseDataDir/geth.log &
+      -targetgaslimit 8000000 -unlock $gethMiningAccount --password <(echo "") -mine &>>$nodeBaseDataDir/geth.log &
```

Comment:
Is there a reason not to set `targetgaslimit` to an even higher number?

Reply:
The current block gas limit on mainnet is 8m so I think it is reasonable match that amount in a dev environment.

Reply:
That's a great reason!